ci-automation/tapfile_helper_lib.sh

```diff
@@ -77,21 +77,25 @@ function __db_init() {
 # 3: <run>     - re-run iteration
 
 function tap_ingest_tapfile() {
+
     local tapfile="${1}"
     local vendor="${2}"
     local run="${3}"
 
     local result=""
     local test_name=""
-    local error_message=""
+    local error_message_file="$(mktemp)"
```

Comment:
The only thing I was wondering about here is whether we should run this function in a subshell and use a trap to remove the temporary file, so we don't litter the tmp filesystem in case of some errors.

Reply:
That's not a bad idea. Will add.